src/highdicom/seg/sop.py

```diff
@@ -2595,12 +2595,9 @@ def segmented_property_categories(self) -> List[CodedConcept]:
             descriptions in this SEG image.
 
         """
-        categories = []
-        for desc in self.SegmentSequence:
-            if desc.segmented_property_category not in categories:
-                categories.append(desc.segmented_property_category)
-
-        return categories
+        return list(
+            {desc.segmented_property_category for desc in self.SegmentSequence}
+        )
```

Comment:
As a set comprehension this may change order, which is undesirable and may additionally break existing user code. Let's revert this to how it was, I don't think this can be achieved with a comprehension

Reply:
I understand. I was under the impression that order is not important here (in which case a set could be returned outright) but that's not documented.

Reply:
Fixed.

Reply:
Thanks. You are right that it is not documented and so users should not be relying on a particular order. Still, it is probably what many will expect and I think it's safer not to change behaviour now for the sake of code neatness